src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,29 @@
+.catalog {
+  --column-count: 1;
```

Comment:
It looks like you declared a CSS variable `--column-count`, but it is not used anywhere in your code. If this variable is not needed, consider removing it.